changelog/9074.improvement.rst

```diff
@@ -0,0 +1,17 @@
+Now ``pytest`` do unpack nested ``pytest.param`` values and merge their ids and marks.
+If parent ``pytest.param`` has an id - it rewrites nested ids.
```

Comment:
I think we can leave this phrase out, seems a bit too detailed for the changelog (this should live in its own docs).

```suggestion

```